### DISCUSSION TITLE
How to download a directory with SFTP?

### DISCUSSION THREAD

**Discussion by ShaiAvr** [isontheline/pro.webssh.net#1222] [Q&A]
I am trying out WebSSh on my MacBook and it works nicely. However, I can't figure out how to download directories to my local machine. When I right-click a file, I get the option to download it, but when I right-click a directory, there is no option to download. How do I download directories to my local machine with WebSSH?

**Comment by isontheline** [comment]
Hello @ShaiAvr 👋

Thank you so much for your feedback 🙏

Ability to download and upload whole folders is not yet available : https://github.com/isontheline/pro.webssh.net/issues/205

I hope to implement it soon however
